test/functional/search_api_test.rb

```diff
@@ -91,14 +92,62 @@ def app
 
      json = JSON.parse(last_response.body)
 
-     assert_equal "/profile/jeff", json['items'][1]['docUrl']
-     assert_equal "jeff",          json['items'][1]['docTitle']
-     assert_equal "user",          json['items'][1]['docType']
+     assert_equal "/profile/steff3",   json['items'][0]['docUrl']
```

Comment:
OK, error 1 here: `NoMethodError: undefined method ‘[]’ for nil:NilClass`

Reply:
It looks like things are not nested the same way as they were in the original test. We should probably conform to the original output format... can you check it against what https://publiclab.org/api/srch/profiles?srchString=steff currently returns?

```
{"items":[{"docId":0,"docType":"user","docUrl":"/profile/iujiu86","docTitle":"iujiu86","docSummary":"","docScore":0}],"srchParams":{"srchString":"steff","seq":null,"showCount":null,"pageNum":null,"tagName":null}}
```

I think to debug this, the best thing to do is to do `puts json` above this line and read it when you hit https://localhost:3000/api/srch/profiles?srchString=admin (or something like that, on your local instance). Make sense? It ought to match the above, and if not, maybe the API output needs to be tweaked?